docs/integrations/maven.md

```diff
@@ -0,0 +1,30 @@
+# Maven
+
+## Recording test results
+
+Apache Maven is a software project management and comprehension tool. See [Maven – Welcome to Apache Maven](https://maven.apache.org/)
+
+After running tests, point to the directory that contains all the generated test report XML files. You can specify multiple directories, for example if you do multi-project build:
```

Comment:
This line says `for example if you do a multi-project build` but the example script includes only one directory. We want to make it clear that multiple directories are supported, so it would be good to update the example.

Reply:
OK, I'll update